types/hand-crafted/JitsiConference.d.ts

```diff
@@ -10,6 +10,8 @@ import JitsiVideoSIPGWSession from './modules/videosipgw/JitsiVideoSIPGWSession'
 import TraceablePeerConnection from './modules/RTC/TraceablePeerConnection';
 import { MediaType } from './service/RTC/MediaType';
 
+type JitsiConferenceEventsType = `${JitsiConferenceEvents}`
```

Comment:
I had to use template literal types here to get it working as expected since the parameter is an enum and we are passing string literals as arguments